src/__tests__/cli.ts

```diff
@@ -0,0 +1,490 @@
+    };
+
+    const appendStderr = (...args: any[]): void => {
+      stderr += args.map((arg) => arg.toString()).join(' ');
+    };
+
+    console.log = appendStdout;
+    console.warn = appendStdout;
+    console.error = appendStderr;
+
+    process.exit = (code: number): never => {
+      exitCode = exitCode ?? code;
+      return undefined as never;
+    };
+
+    process.chdir(testProjectDir);
+
+    await main({ init, flow, update });
+
+    return { exitCode: exitCode ?? 0, stdout, stderr };
+  } finally {
+    process.chdir(originalCwd);
+    process.exit = originalExit;
+    Object.entries(originalConsole).forEach(([key, value]) => {
+      console[key] = value;
+    });
+  }
+}
+
+async function createProject(
+  files: Array<{ name: string; content: string }>,
+  baseDir = '.',
+): Promise<string> {
+  const randomId = Math.floor(Math.random() * 1000000);
+
+  const testSpaceDir = path.join('.test-space', randomId.toString());
+
+  await mkdir(testSpaceDir, { recursive: true });
+
+  await Promise.all(
+    files.map((file) =>
+      mkdir(path.join(testSpaceDir, path.dirname(file.name)), {
+        recursive: true,
+      }),
+    ),
+  );
+
+  await Promise.all(
+    files.map((file) =>
+      writeFile(path.join(testSpaceDir, file.name), file.content),
+    ),
+  );
+
+  return path.join(testSpaceDir, baseDir);
+}
+
+cases(
+  'cli integration tests',
+  async (scenario) => {
+    const testProjectDir = await createProject(
+      scenario.files,
+      scenario.baseDir,
+    );
+
+    try {
+      const { stdout, stderr, exitCode } = await exec(testProjectDir);
+
+      expect(stdout).toMatch(scenario.stdout);
+      expect(stderr.replace(/- initializing\s+/, '')).toMatch('');
+      expect(exitCode).toBe(scenario.exitCode);
+    } finally {
+      await rmdir(testProjectDir, { recursive: true });
+    }
+  },
+  [
+    {
+      name: 'should identify unimported file',
+      files: [
+        { name: 'package.json', content: '{ "main": "index.js" }' },
+        { name: 'index.js', content: `import foo from './foo';` },
+        { name: 'foo.js', content: '' },
+        { name: 'bar.js', content: '' },
+      ],
+      exitCode: 1,
+      stdout: /1 unimported files.*bar.js/s,
+    },
+    {
+      name: 'should identify unresolved imports',
+      files: [
+        { name: 'package.json', content: '{ "main": "index.js" }' },
+        { name: 'index.js', content: `import foo from './foo';` },
+      ],
+      exitCode: 1,
+      stdout: /1 unresolved imports.*.\/foo/s,
+    },
+    {
+      name: 'should identify unimported file in meteor project',
+      files: [
+        {
+          name: 'package.json',
+          content:
+            '{ "meteor" : { "mainModule": { "client": "client.js", "server": "server.js" } } }',
+        },
+        { name: 'client.js', content: `import foo from './foo';` },
+        { name: 'server.js', content: '' },
+        { name: '.meteor', content: '' },
```

Comment:
This will create a file named `.meteor` correct? In meteor projects, that's a directory. 

```suggestion
        { name: '.meteor/release', content: 'METEOR@1.11.1' },
```

Reply:
That's correct. 

Your suggestion will create a directory called `meteor` with a single file, `release` (no extension) with the content `METEOR@1.11.1` (no quotes). 

Is that what you intended?

Reply:
Yes, that's what I intended. But I chose to edit the suggestion, and clear the contents. To make it more "timeless". We don't use the `meteor release` version in our code, so it doesn't make sense to populate it.

Now, it creates a directory `.meteor` with a single empty file named `release`. That's fine.